templates/batch-llm/README.ipynb

```diff
@@ -371,7 +387,25 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Summary\n",
+    "## Submitting an Anyscale Job\n",
```

Comment:
Remove this for now, the jobs tutorial isn't ready yet. When it is, we can link to that instead of repeating the same content in each template.

Reply:
Also, this will be "ray job submit" within workspaces.